pages/company/index.tsx

```diff
@@ -555,7 +558,17 @@ const Index = ({
                 </div>
               </ContentOverImage>
               <div className={`${Landing["news"]} section`}>
-                <News items={news} />
+                <div className={clsx(Grid.container, Grid.margin_center)}>
+                  <NewsGrid news={news} title="Latest News" />
+                  <Button
+                    style={{ display: "block" }}
+                    onClick={() => {
+                      router.push(newsroomPageSettings.path);
+                    }}
+                  >
+                    View All
```

Comment:
```suggestion
                    View all
```

Sentence casing. Use css if it has to be all upper case in a button.